src/Schema/Grammar.php

```diff
@@ -420,16 +420,18 @@ public function compileCreateChangeStream(Blueprint $blueprint, ChangeStreamDefi
     /**
      * @param Blueprint $blueprint
      * @param ChangeStreamDefinition $command
-     * @return string
+     * @return list<string>
      */
-    public function compileAlterChangeStream(Blueprint $blueprint, ChangeStreamDefinition $command): string
+    public function compileAlterChangeStream(Blueprint $blueprint, ChangeStreamDefinition $command): array
     {
         $parts = [];
-        $parts[] = "alter change stream {$this->wrap($command->stream)}";
+        if ($command->tables !== []) {
```

Comment:
if you want to change to `set for all` then whether `$command->tables` is empty, ddl should be create.